### ISSUE TITLE
How do I retrive unauthenticated Amazon Cognito Identity

### ISSUE THREAD

**Issue by aeggbeg** [aws/aws-sdk-js-v3#2427]
Hi,
I am following this document - https://docs.aws.amazon.com/cognito/latest/developerguide/getting-credentials.html which uses SDK-v2 and the following code

`var identityId = AWS.config.credentials.identityId;`
I am using below V3 code

```
const { CognitoIdentityClient } = require("@aws-sdk/client-cognito-identity");
const { fromCognitoIdentityPool } = require("@aws-sdk/credential-provider-cognito-identity");
const { Kinesis, PutRecordsCommand } = require("@aws-sdk/client-kinesis");

const REGION = "us-east-1";
const client = new Kinesis({
  region: REGION,
  credentials: fromCognitoIdentityPool({
    client: new CognitoIdentityClient({region: REGION}),
    identityPoolId: "us-east-1:XXXX-XXX-XXX-XXXX" 
    
  })
});
```
How do I retrive unauthenticated Amazon Cognito Identity in a variable? V2 and V3 is completely different and it's hard to follow.
Thank you for your help.

**Comment by ajredniwja** [comment]
Hi @aeggbeg thanks for opening this issue, you can do something like:

```JavaScript
    const client = new CognitoIdentityClient({ region });
    const command = new GetIdCommand({IdentityPoolId: idPool});
    const response = await client.send(command);        
    console.log(response.IdentityId)
```

Closing this issue now, please reach out if you have any other questions.
